### PR TITLE
Fix data race in namespace error handling

### PR DESCRIPTION
**Pull request by gaurav1086**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
/kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Fix data race in the error handling.
**Which issue(s) this PR fixes**:

Fixes #
N/A
**Special notes for your reviewer**:
None
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: test/e2e/apimachinery/namespace.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func extinguish(f *framework.Framework, totalNS int, maxAllowedAfterDel int, max
 			defer wg.Done()
 			defer ginkgo.GinkgoRecover()
 			ns := fmt.Sprintf("nslifetest-%v", n)
-			_, err = f.CreateNamespace(ns, nil)
+			_, err := f.CreateNamespace(ns, nil)
```
Comment (liggitt) [inline test/e2e/apimachinery/namespace.go:52]: remove `var err error` above?

Comment (gaurav1086) [reply test/e2e/apimachinery/namespace.go:50]: Thanks for pointing out. Code changes done.

**Comment by liggitt** [comment]
/approve

thanks, please squash to a single commit, then lgtm

**Comment by gaurav1086** [comment]
@liggitt commits squashed. Thank you.

**Pull request merged**